src/scripting/lua_map.cc

```diff
@@ -6713,7 +6713,7 @@ int LuaMarket::propose_trade(lua_State* L) {
 	const Widelands::BillOfMaterials items_to_send =
 	   parse_wares_as_bill_of_material(L, 4, self->owner().tribe());
 	// TODO(sirver,trading): unsure if correct. Test inter-tribe trading, i.e.
-	// barbarians trading with empire, but shipping atlantean only wares.
+	// Barbarians trading with empire, but shipping atlantean only wares.
```

Comment:
```suggestion
	// Barbarians trading with Empire, but shipping Atlantean only wares.
```
:smiley:

Reply:
This PR was only about the Barbarians 😄 , but I'll accept those suggestions preparing for the next ones